toolkits/jira/arcade_jira/utils.py

```diff
@@ -38,7 +39,7 @@ def is_valid_date_string(date_string: str) -> bool:
 
 
 def quote(v: str) -> str:
-    return f'"{v.replace('"', '\\"')}"'
+    return repr(v)  #  f'"{v.replace('"', '\\"')}"'
```

Comment:
What is the intent of this function?

Function is named `quote` and _repr() is a built-in function in Python used to obtain the string representation of an object_

Reply:
good catch, this change should have been reverted before that commit